docs/providers/aws/guide/serverless.yml.md

```diff
@@ -16,115 +16,272 @@ layout: Doc
 
+## Root properties
+
 ```yml
 # serverless.yml
 
-service: myService
+# Service name
+service: myservice
 
+# Framework version constraint: '3', '>=2.33', '^2.33'
 frameworkVersion: '3'
-configValidationMode: warn # Modes for config validation. `error` throws an exception, `warn` logs error to console, `off` disables validation at all. The default is warn.
-enableLocalInstallationFallback: false # If set to 'true', guarantees that it's a locally (for service, in its node_modules) installed framework which processes the command
-useDotenv: false # If set to 'true', environment variables will be automatically loaded from .env files
-deprecationNotificationMode: warn:summary # 'warn' reports deprecations on the go, 'error' will result with an exception being thrown on first approached deprecation
 
-disabledDeprecations: # Disable deprecation logs by their codes. Default is empty.
+# Configuration validation: 'error' (fatal error), 'warn' (logged to the output) or 'off' (default: warn)
+# See https://www.serverless.com/framework/docs/configuration-validation
+configValidationMode: error
+# Run the local 'serverless' package if it exists in 'node_modules'
+enableLocalInstallationFallback: false
+# Load environment variables from .env files (default: false)
+# See https://www.serverless.com/framework/docs/environment-variables
+useDotenv: true
+# 'warn' reports deprecations on the go, 'error' will result with an exception being thrown on first approached deprecation
+deprecationNotificationMode: warn:summary
+# Disable deprecations by their codes (default: empty)
+# See https://www.serverless.com/framework/docs/deprecations
+disabledDeprecations:
   - DEP_CODE_1 # Deprecation code to disable
   - '*' # Disable all deprecation messages
+```
+
+### Parameters
+
+Learn more about stage parameters in the [Parameters documentation](../../../guides/parameters.md).
+
+```yml
+# serverless.yml
+
+# Stage parameters
+params:
+  # Values for the "prod" stage
+  prod:
+    my-parameter: foo
+  # Values for the "dev" stage
+  dev:
+    my-parameter: bar
+```
+
+## Provider
+
+### General settings
+
+```yml
+# serverless.yml
 
 provider:
   name: aws
+  # Default stage (default: dev)
+  stage: dev
+  # Default region (default: us-east-1)
+  region: us-east-1
+  # The AWS profile to use to deploy
+  profile: production
+  # Use a custom name for the CloudFormation stack
+  stackName: custom-stack-name
+  # Use a custom name for the API Gateway API
+  apiName: custom-api-name
+  # Use a custom name for the websockets API
+  websocketsApiName: custom-websockets-api-name
+  # custom route selection expression
+  websocketsApiRouteSelectionExpression: $request.body.route
+  # optional, if set to 'true', disables default behavior of generating export names for CloudFormation outputs
+  disableDefaultOutputExportNames: false
+  # Optional CloudFormation tags to apply to all services and functions
```

Comment:
That is a bit misleading - we only apply these tags to API GW and functions and nothing else

Reply:
Thanks, I clarified that